code/_globalvars/lists/engineering.dm

```diff
@@ -0,0 +1,23 @@
+// List of all tool behaviours.
+GLOBAL_LIST_INIT(all_tool_behaviours, list(
+	TOOL_ANALYZER,
```

Comment:
For bonus points replace `/omnitool/get_all_tool_behaviours()` with `return GLOB.all_tool_behaviours)`

Reply:
I already did